internal/language/language.go

```diff
@@ -128,11 +126,11 @@ func MakeByName(name string) Language {
 // MakeByNameAndVersion will retrieve a language by a given name and version.
 func MakeByNameAndVersion(name, version string) (Language, error) {
 	if strings.ToLower(name) == Python2.Requirement() {
-		version, err := semver.Parse(version)
-		if err != nil {
-			return Unknown, err
+		if len(version) == 0 {
+			return Unknown, locale.NewError("err_language_make_version", "Invalid version number, should be of format <major>.<minor>.<patch>")
 		}
-		name = name + strconv.FormatUint(version.Major, 10)
+		parts := strings.Split(version, ".")
+		name = name + parts[0]
```

Comment:
This is too specific to our current use-case. Instead let's just drop everything after 0.0.0 and send that to semver.

eg. `1.2.3.4` becomes `1.2.3`.